.gitmodules

```diff
@@ -19,9 +19,11 @@
 [submodule "contracts/lib/core"]
 	path = contracts/lib/core
 	url = https://github.com/symbioticfi/core
+	tag = 1.0.0-devnet.10
```

Comment:
I don't think a .gitmodules file supports `tag` like this. See https://git-scm.com/docs/gitmodules. What are you attempting to do here?

Reply:
Yeah looks like it.